.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+### Are there any dependencies that need to be addresses before this PR is merged?
+
+
+### What ticket(s) or other PRs does this relate to?
+
+
+### What was the problem or feature?
+
+
+### What was the solution?
+
+
+### Where does this work fall on the Good - Fast spectrum?
```

Comment:
Is this to address the completeness of the solution or the urgency to review and publish?

Reply:
Oh, that's a good idea. WOuld it be worth asking two questions there?

Reply:
Primarily the completeness of the solution - particularly now as we try to build quickly, this is a space to call out specific technical corners that were cut in the name of expediency or that the solution aims for completeness. This gives the author the opportunity to self-evaluate and/or justify the coding decisions they've made, and gives the reviewer more context into the scope and intent of the change.

Reply:
(As for the "urgency to review", I think that should be communicated through other channels (slack, review requests, etc), as the description of a PR probably isn't the best place to stick a desperate plea for reviews of said PR)